libs/chat-workflow/src/main/java/org/finos/springbot/workflow/help/HelpController.java

```diff
@@ -75,8 +73,17 @@ private boolean includeInHelp(ChatMapping<ChatRequest> hm) {
 	private CommandDescription convertToCommandDescriptions(ChatMapping<ChatRequest> hm) {
 		ChatRequest e = hm.getMapping();
 		ChatHandlerMethod m = hm.getHandlerMethod();
-		String d = StringUtils.hasText(e.description()) ? e.description() : defaultDescription(m.getMethod());
-		return new CommandDescription(e.isButtonOnHelpPage(), e.value()[0], d, Arrays.asList(e.value()));
+		String description = StringUtils.hasText(e.description()) ? e.description() : defaultDescription(m.getMethod());
+
+		boolean addButton = e.isButtonOnHelpPage();
+		if(addButton) {
```

Comment:
This logic should be inside the "includeInHelp" function, otherwise you'll get CommandDescriptions added to the list that shouldn't be there.